src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java

```diff
@@ -128,6 +134,8 @@ CourtCaseResponse getCourtCase(
     public @ResponseBody
     Mono<ExtendedHearingRequestResponse> createOrUpdateHearingByHearingId(@PathVariable(value = "hearingId") String hearingId,
                                                                           @Valid @RequestBody ExtendedHearingRequestResponse putHearingRequest) {
+
+        log.info("processed by ApplicationContext " + applicationContext.getId());
```

Comment:
simply to show that both application contexts are handling the requests - remove before merging if we decide to go with this.